src/DepositHandler.sol

```diff
@@ -19,7 +19,8 @@ contract DepositHandler is Pausable, AccessControl, IDepositHandlerErrors {
     uint256 public immutable bootcampDuration;
     uint256 public immutable bootcampStartTime;
     IERC20 public immutable depositToken;
-    bool allowEmergencyWithdraw;
+    address[] public emergencyWithdrawParticipants;
```

Comment:
There may be an optimization, let's see later

Reply:
+